src/gui/folder.cpp

```diff
@@ -1297,6 +1330,7 @@ void Folder::registerFolderWatcher()
         this, &Folder::slotNextSyncFullLocalDiscovery);
     connect(_folderWatcher.data(), &FolderWatcher::becameUnreliable,
         this, &Folder::slotWatcherUnreliable);
+    connect(_folderWatcher.data(), &FolderWatcher::filesLockReleased, this, &Folder::slotFilesLockReleased);
```

Comment:
should you not first check if the server has the capability to do temp files lock ?
I would probably not execute the connect f the server has no support for temp files lock
same comment for the `FolderWatcher` feature